src/assets/i18n/fr.json

```diff
@@ -34,14 +34,14 @@
     "Nameservers": "Serveurs DNS",
     "No data for this test": "Il n'y a pas de résultat pour ce test",
     "No result for this query": "Il n'y a pas de résultat pour ce test",
```

Comment:
Both the same text for those translations, is that intended?

Reply:
:shrug: